spark-operator-docs/spark_application.md

```diff
@@ -0,0 +1,221 @@
+## Spark Application API
```

Comment:
Please remove all `spark-operator-docs` from this PR. We didn't start it yet, @jiangzho .